src/fxos/index.js

```diff
@@ -0,0 +1,13 @@
+let modularize = require('../modularize');
+
+let Fxos = modularize(function Fxos(adb) {
```

Comment:
So the individual endpoints have access to a `req` with an `adb` tacked on -- I was thinking that we could have the `Fxos` helpers that need take an `adb` param?